package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "gulp-jade",
+  "name": "gulp-pug",
```

Comment:
might need to stay gulp-jade but support pug if the person that squatted the gulp-pug name doesn't want to collaborate.